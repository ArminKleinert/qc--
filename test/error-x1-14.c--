
register bits8 c;

section "data" 
{
    x:
    bits32[4] { 1, 2, 3, 4, 5 };

}

