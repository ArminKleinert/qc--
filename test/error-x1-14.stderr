file error-x1-14.c--, line 7, characters 5-33 Error: too many initial values
